apps/core-app/src/main/modules/box-tool/addon/apps/win.test.ts

```
@@ -156,6 +156,33 @@ describe('win app scanner', () => {
     })
   })
 
+  it('treats absolute Get-StartApps AppId values as desktop apps', async () => {
+    const appPath = 'D:\\Tools\\Codex\\Codex.exe'
+
+    readdirMock.mockResolvedValue([])
+    statMock.mockImplementation(async (target: string) => {
+      if (target === appPath) return createFileStat()
```

ESLint

[error] Expect newline after if

(antfu/if-newline)